.travis.yml

```
@@ -22,30 +16,38 @@ jobs:
       before_install:
         - HOMEBREW_NO_AUTO_UPDATE=1 brew install ccache
       env: &Release
-        - TRAVIS_CUBERITE_BUILD_TYPE=RELEASE
+        - TRAVIS_CUBERITE_BUILD_TYPE=Release
 
     - name: "AppleClang - Debug"
       os: osx
       osx_image: xcode11.3
       before_install:
         - HOMEBREW_NO_AUTO_UPDATE=1 brew install ccache
       env: &Debug
-        - TRAVIS_CUBERITE_BUILD_TYPE=DEBUG
+        - TRAVIS_CUBERITE_BUILD_TYPE=Debug
 
     - name: "Clang 7.0 - Release"
       compiler: clang
       before_install: &use-cmake
-        - export PATH=/snap/bin/:${PATH}
+        - wget --output-document=${HOME}/CMake http://cmake.org/files/v3.17/cmake-3.17.2-Linux-x86_64.tar.gz
+        - tar --extract --one-top-level=${HOME}/SeeMake --strip-components 1 --file ${HOME}/CMake
+        - export PATH=${HOME}/SeeMake/bin/:${PATH}
       env: *Release
 
     - name: "Clang 7.0 - Debug"
       compiler: clang
       before_install: *use-cmake
       env: *Debug
 
-    - name: "GCC 7.4 - Release, CMake 3.12"
+    - name: "GCC 7.4 - Release, CMake 3.12, jumboless"
       compiler: gcc
-      env: *Release
+      before_install:
+        - wget --output-document=${HOME}/CMake http://cmake.org/files/v3.13/cmake-3.13.0-rc1-Linux-x86_64.tar.gz
```

That's a pre-release version.
```suggestion
        - wget --output-document=${HOME}/CMake https://github.com/Kitware/CMake/releases/download/v3.13.0/cmake-3.13.0-Linux-x86_64.tar.gz
```